blog/_posts/2025-08-14-node-autoprovisioning.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Announcement - Node Auto Provisioning"
+description: "Learn about Node Auto Provisioning for Azure Kubernetes Service, based on Karpenter, and how it can simplify the scaling experience for your workloads on AKS."
+date: 2025-08-14
+author: Wilson Darko # must match the authors.yml in the _data folder
+categories: 
+- general, add-ons, compute
+---
+
+We’re excited to announce the General Availability (GA) of Node Auto Provisioning (NAP) for Azure Kubernetes Service (AKS)!
```

Comment:
Start / lead with the problem statement for Why NAP came into being - what was the status before this .  Who  were the customers / personas requesting for this and why .